docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Since it's related to `dev/Dockerfile`, can `docker-entrypoint` be moved into that dir as well?